src/SmartComponents/AvailableTasks/AvailableTasks.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+const AvailableTasks = () => {
+  return (
+    <React.Fragment>
+      <div aria-label="available-tasks">Available tasks</div>
+    </React.Fragment>
+  );
+};
+
+export default withRouter(AvailableTasks);
```

Comment:
`withRouter` is and older paradigm of react-router-dom which has been removed in latest major version (v6), they instead recommend using Hooks;

also maybe it's just a placeholder, but it does not do anything in the current code

[useHistory](https://v5.reactrouter.com/web/api/Hooks/usehistory)
[useLocation](https://v5.reactrouter.com/web/api/Hooks/uselocation)
[useParams](https://v5.reactrouter.com/web/api/Hooks/useparams)
[useRouteMatch](https://v5.reactrouter.com/web/api/Hooks/useroutematch)

Reply:
Yes, these will be placeholders for now. I may remove them in a future PR.